core/src/test/java/tachyon/client/TachyonFSTest.java

```diff
@@ -56,10 +56,8 @@ public final void before() throws IOException {
   }
 
   @AfterClass
-  public static void afterClass() throws Exception {
+  public static final void afterClass() throws Exception {
```

Comment:
Seems like most of our `afterClass()` are `public static final` I don't see any reason why they can't just be `public static`.